rust/BUILD.md

```diff
@@ -75,6 +75,11 @@ cargo build --features mimalloc,stack-array
 
 1. Identify a possible optimization opportunity by
     - running the Go VM benchmarks and comparing in which cases `evmrs` is slower than the other interpreters
+        ```sh
+        # assuming you are in Tosca
```

Comment:
Simply this, can we run it from the Root directory?

Reply:
I've now changed all scripts s.t. they are expected to be run from Tosca/rust.

Reply:
I see, I have to say that I am quite worried about the escalation of scripts and different procedures for the rust project. I understand the use, but I believe most of it is scaffolding needed for early development and not functional parts of the project.
Maybe we should sit down some day and explore what options do we have with the goal of:
- use all the tools we can to assist the task
- prevent the escalation of scaffolding scripts and obscure tools to facilitate reviews and define the boundaries of what is the final product. 

Usually most of the scripting you have been adding here I add it to vscode tasks, I keep private my workflows because they involve temporary files, user folders, and other procedures which other developpers may not use.